components/dropdown/dropdown-content-mixin.js

```diff
@@ -51,6 +51,16 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 				reflect: true,
 				attribute: 'no-padding'
 			},
+			noPaddingFooter: {
+				type: Boolean,
+				reflect: true,
+				attribute: 'no-padding-footer'
+			},
+			noPaddingHeader: {
+				type: Boolean,
+				reflect: true,
+				attribute: 'no-padding-header'
+			},
```

Comment:
When time-input applied this attribute, the analyzer correctly complained that the property didn't actually exist... which is true -- there was only CSS for it.